FIRO_TSEnsembles/src/main/java/hec/metrics/MetricCollection.java

```diff
@@ -0,0 +1,62 @@
+package hec.metrics;
+
+import hec.stats.Configuration;
+
+import java.time.Duration;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class MetricCollection {
+    public MetricCollectionTimeSeries parent;
+    private Configuration _configuration;
+    private float[][] metrics;
+    private String[] metric_statistics;
+
+    public MetricCollection(Configuration c, String[] statistics)
+    {
+        this._configuration = c;
+        this.metric_statistics = statistics;
+    }
+    public MetricCollection(ZonedDateTime issueDate,ZonedDateTime startDate, String[] statistics)
+    {
+        this(new MetricsConfiguration(issueDate,startDate), statistics);
+    }
+
+    public MetricCollection(ZonedDateTime issueDate, ZonedDateTime startDate, float[][] ensemble, String[] parameters) {
+    }
+    public int parameterIndex(String parameterName){
```

Comment:
I can't see parameterIndex(.. ) called anywhere, can we make it private or protected?

Reply:
noted - it is a work in progress, i am planning a refactor around this stuff today.